src/services/user.ts

```diff
@@ -1,9 +1,10 @@
 import axios from 'axios'
 import { User } from '../types/User'
+const SERVER_URL = process.env.REACT_APP_SERVER_URL
```

Comment:
Just a very tiny improvement: you could have a `constants` file in the `src` folder and then export the `SERVER_URL`, instead of making all the services aware about from where to get it

```javascript
export const SERVER_URL = process.env.REACT_APP_SERVER_URL
```

Reply:
@leo-diehl do you think it's better to put it inside the folder utils or src?